asm/entities_1.s

```diff
@@ -3075,7 +3075,7 @@ _080129F8:
 	movs r3, #0xc0
 	lsls r3, r3, #0x12
 	adds r3, r6, r3
-	ldr r1, _08012A88 @ =gUnknown_080D672C
+	ldr r1, _08012A88 @ =PlayerCharIdleAnims
```

Comment:
I'm personally not a fan of `Char` because of the confusion with `char`. I think we can just replace `PlayerChar` for `Character`? As what other characters are there in this game?

Reply:
I mean, it's not a big deal to rename it, but I think the rest of the name gives enough context, right? :D